classes/Rest/Controllers/AppKernelControllerProvider.php

```diff
@@ -1628,13 +1617,16 @@ public function getRawPerformanceMap(Request $request, Application $app)
 
         $data = array();
         try {
-            $perfMap = new \AppKernel\PerformanceMap(array(
-                'start_date' => $startDate,
-                'end_date' => $endDate,
-                'resource' => array('data' => $user->getResources()),
-                'appKer' => $appKernels,
-                'problemSize' => $problemSizes
-            ));
+            $options = array(
+                    'start_date' => $startDate,
+                    'end_date' => $endDate,
+                    'appKer' => $appKernels,
+                    'problemSize' => $problemSizes
+            );
+            if (!$user->hasAcl(ROLE_ID_PROGRAM_OFFICER)) {
+                $options['resource'] = array('data' => $user->getResources());
+            }
```

Comment:
Why does this apply to everyone but program officers?  (I'm assuming that program officers should have access to all resources, but does `$user->getResources()` not return all resources for program officers?)